File: (7 kyu) Build a square/(7 kyu) Build a square.py
# #1
# def generateShape(int):
#     str = []
#     for i in range(int):
#         str.append('+' * int)
#     return "\n".join(str)
# #2
# def generateShape(n):
#     return "\n".join("+" * n for i in range(n))


def generateShape(int):  # 3
    return (("+" * int + "\n") * int)[:-1]

File: (7 kyu) Build a square/(7 kyu) Build a square.cs
using System;

public static class Kata
{
  public static string GenerateShape(int n)
  {
    string[] res = new string[n];
    for (int i = 0; i < n; i++)
    {
      res[i] = new string('+', n);
    }
    return String.Join("\n", res);
  }
}
